backend/benefit/calculator/api/v1/serializers.py

```diff
@@ -272,6 +272,15 @@ def validate(self, data):
                 raise serializers.ValidationError(
                     {"end_date": _("End date cannot be empty")}
                 )
+            calculation = self.context["request"].data["calculation"]
```

Comment:
Seems that `request` is already in the context and `data` also maybe?

Reply:
<img width="310" alt="Screenshot 2023-06-13 at 11 48 25" src="https://github.com/City-of-Helsinki/yjdh/assets/33894149/6c295d1c-6e52-4c4f-b8f8-3d01b33f78ad">

Reply:
In this stage of executio, data does not have any knowledge of the calculation.